AppCenterAnalytics/AppCenterAnalytics/Internals/Model/MSEventLog.m

```diff
@@ -89,7 +96,7 @@ - (MSCommonSchemaLog *)toCommonSchemaLogForTargetToken:(NSString *)token {
       NSMutableDictionary *destProperties = csProperties;
       for (NSUInteger i = 0; i < lastIndex; i++) {
         NSMutableDictionary *subObject = nil;
-        if ([(NSObject *)destProperties[csKeys[i]] isKindOfClass:[NSMutableDictionary class]]) {
+        if ([(NSObject *) destProperties[csKeys[i]] isKindOfClass:[NSMutableDictionary class]]) {
```

Comment:
Why not write out the variable names? Chars are cheap ;)

Reply:
Perhaps a good change, but i don't want to touch this code (it was just reformatted)